backend/src/blockchain/GradidoNodeRequests.ts

```diff
@@ -0,0 +1,43 @@
+import CONFIG from '../config'
+import { RpcClient } from 'jsonrpc-ts'
+import { randomInt } from 'crypto'
+
+interface GroupDetails {
+  GroupName: string
+  GroupAlias: string
+  CoinColor: number | string
```

Comment:
Hu? What is this for?

Reply:
On Blockchain-Side I have already implemented the coinColor concept Bernd talked about. So every transaction has also a coin color.
So user can decide by themself which coins they would like to accept. 
For that to work I need on the blockchain the info which group has which native coin color.
The Gradido-Node has a json-rpc function to get the GroupDetails for a group alias. For jsonrpc-ts I must define the returned data type. 
In my test Setup the Apollo Server uses this to check if his group was already registered and if not create a transaction for that. It isn't ideal, but enough for tests. Later it can be implemented in admin so that the Server-Admin can manually send this register group transaction as part of the server setup.